openmetadata-ui/src/main/resources/ui/src/utils/DomainUtils.tsx

```diff
@@ -215,12 +215,11 @@ export const renderDomainLink = (
 ) => (
   <Link
     className={classNames(
-      'no-underline domain-link',
+      'no-underline',
```

Comment:
I guess we were using domain-link heavily can you check if it's not impacting anything else